### DISCUSSION TITLE
Questions about Euphony's range of sound waves

### DISCUSSION THREAD

**Discussion by HongEunbeen** [euphony-io/euphony#149] [Q&A] [answered]
Hi🙌

I'm thinking about the idea of ​​a sample app using a Euphony's library, and I'm curious how the range of sound waves is determined.

How far can the receiver receive the sender's data?
Can a developer or user set  range of sound waves throgth a library?

Is it possible to measure the distance by the timestamp of sending and receiving? 🙏

**Comment by zion830** [comment, marked as answer]
* When I personally tested it, it was smooth up to about 1-2m
* The proper distance has changed depending on the volume of the device
* If the volume is low, you have to place the devices close to each other

**Reply by HongEunbeen** [reply]
Thank you for answer. I think it would be nice to have a document about sound waves  distance..!!🤔

**Reply by designe** [reply]
I think this answer is the most appropriate! :)

Euphony uses high notes in sound waves, it has a straight-forward characteristic.
Therefore, the recognition range varies depending on the direction of the speaker.

As @zion mentioned, ~2m is typical, and if you take advantage of the straight-line characteristic, you can send it up to 5m.

As you know, since it's a sound wave, you can adjust the transmission range by adjusting the volume.

**Comment by Kimhaedam** [comment]
I remember that the maximum distance is about 5m at the launching ceremony. However, as @zion830 answered, it seems to be slightly different depending on the setting of the devices and the environment of the test space.

**Comment by win-luck** [comment]
@Kimhaedam @zion830 How could you know about the maximum distance..?

**Reply by Kimhaedam** [reply]
When the remaining people went to the study center after the launching ceremony, briefly ran the Euphony, and I heard it then.😀
